vdom/src/com/vdom/core/Game.java

```diff
@@ -2830,8 +2814,9 @@ protected void initCards() {
         	if (obeliskCard == null) {
         		ArrayList<Card> validObeliskCards = new ArrayList<Card>();
             	for (String pile : piles.keySet()) {
-            		if (piles.get(pile).isSupply() && piles.get(pile).card().is(Type.Action)) {
-            			validObeliskCards.add(piles.get(pile).card());
+                    //TODO SPLIT PILES set obelistk to whole pile, and placeholder
```

Comment:
Just set the Obelisk card to be the card's pile's placeholder instead and have all checks if something is an Obelisk card check that the piles of both cards are equal.